class-wc-facebookcommerce.php

```diff
@@ -108,6 +112,7 @@ public function init() {
 				$this->product_feed            = new \SkyVerge\WooCommerce\Facebook\Products\Feed();
 				$this->products_sync_handler   = new \SkyVerge\WooCommerce\Facebook\Products\Sync();
 				$this->sync_background_handler = new \SkyVerge\WooCommerce\Facebook\Products\Sync\Background();
+				$this->product_feed = new \SkyVerge\WooCommerce\Facebook\Products\Feed();
```

Comment:
Duplicated.

Reply:
Thank for catching that one. I removed the duplicated line in 6664e0d.